README.md

```diff
@@ -32,10 +22,23 @@ We simulate Test-Driven Development (TDD) by implementing the tests in order of
 Test files should use the following format:
 
 ```
-# include the body of an example test
+  test("says hello world with no name", () {
+    final String result = helloWorld.hello();
+    expect(result, equals("Hello, World!"));
+  }, skip: false);
```

Comment:
What's the purpose of this `skip: false`? The default behavior is not to skip a test, so this is redundant.

Reply:
Hm. My thinking is that `skip: false` made it more clear to a user how to enable the other tests (change `true` to `false`). But deleting `, skip: false` is also possibly obvious.

How do other tracks handle this? Is there a consensus or established wisdom?

Reply:
The automated testing switches this flag to true and then back to false after testing. Although I don't understand its purpose either.

Reply:
It's so users focus on one failing test versus being told all the tests are failing.

Users are meant to solve the exercise one test at a time.

Now it's true, the `skip: false` doesn't need to be there, if that's what people were confused about.

Reply:
@Stargator Ah, I see what you mean. Initially, users will skip all tests but the first one, and work they way through each test individually, to get them used to TDD. Alright, that sounds good to me.